contracts/src/op/OPSuccinctDisputeGame.sol

```diff
@@ -0,0 +1,362 @@
+
+    /// @notice The game type ID.
+    GameType internal immutable GAME_TYPE;
+
+    /// @notice The anchor state registry.
+    OPSuccinctAnchorStateRegistry internal immutable ANCHOR_STATE_REGISTRY;
+
+    /// @notice The chain ID of the L2 network this contract argues about.
+    uint256 internal immutable L2_CHAIN_ID;
+
+    /// @notice The index of the block number in the RLP-encoded block header.
+    /// @dev Consensus encoding reference:
+    /// https://github.com/paradigmxyz/reth/blob/5f82993c23164ce8ccdc7bf3ae5085205383a5c8/crates/primitives/src/header.rs#L368
+    uint256 internal constant HEADER_BLOCK_NUMBER_INDEX = 8;
+
+    /// @notice Semantic version.
+    /// @custom:semver 1.0.0
+    string public constant version = "1.0.0";
+
+    /// @notice The timestamp of the game's global creation.
+    Timestamp public createdAt;
+
+    /// @notice The timestamp of the game's global resolution.
+    Timestamp public resolvedAt;
+
+    /// @notice Returns the current status of the game.
+    GameStatus public status;
+
+    /// @notice Flag for the `initialize` function to prevent re-initialization.
+    bool internal initialized;
+
+    /// @notice Flag for whether or not the L2 block number claim has been invalidated via `challengeRootL2Block`.
+    bool public l2BlockNumberChallenged;
+
+    /// @notice The challenger of the L2 block number claim. Should always be `address(0)` if `l2BlockNumberChallenged`
+    ///         is `false`. Should be the address of the challenger if `l2BlockNumberChallenged` is `true`.
+    address public l2BlockNumberChallenger;
+
+    /// @notice Claim made by the proposer.
+    ClaimData public claimData;
+
+    /// @notice Credited balances for winning participants.
+    mapping(address => uint256) public credit;
+
+    /// @notice The latest finalized output root, serving as the anchor for output bisection.
+    OutputRoot public startingOutputRoot;
+
+    /// @param _absolutePrestate The absolute prestate of the instruction trace.
+    /// @param _maxClockDuration The maximum amount of time that may accumulate on a team's chess clock.
+    /// @param _anchorStateRegistry The contract that stores the anchor state for each game type.
+    /// @param _l2ChainId Chain ID of the L2 network this contract argues about.
+    constructor(
+        Claim _absolutePrestate,
+        Duration _maxClockDuration,
+        // IAnchorStateRegistry _anchorStateRegistry,
+        OPSuccinctAnchorStateRegistry _anchorStateRegistry,
+        uint256 _l2ChainId
+    ) {
+        // Set up initial game state.
+        ABSOLUTE_PRESTATE = _absolutePrestate;
+        MAX_CLOCK_DURATION = _maxClockDuration;
+        // TODO: Once a new version of the Optimism contracts containing the PR below is released,
+        // update this to return the correct game type: GameTypes.OP_SUCCINCT
+        // https://github.com/ethereum-optimism/optimism/pull/13780
+        GAME_TYPE = GameType.wrap(6);
```

Comment:
Btw, we'll need a different `GAME_TYPE` for this (as it's a different game than the full-validity integration with the dispute game).

Use GameType.wrap(7) for now?